index.js

```diff
@@ -261,11 +262,15 @@ async function fastifyHttpProxy (fastify, opts) {
   }
 }
 
-fastifyHttpProxy[Symbol.for('plugin-meta')] = {
+module.exports = fp(fastifyHttpProxy, {
   fastify: '4.x',
   name: '@fastify/http-proxy'
-}
-
-module.exports = fastifyHttpProxy
-module.exports.default = fastifyHttpProxy
-module.exports.fastifyHttpProxy = fastifyHttpProxy
+})
+module.exports.default = fp(fastifyHttpProxy, {
+  fastify: '4.x',
+  name: '@fastify/http-proxy'
+})
+module.exports.fastifyHttpProxy = fp(fastifyHttpProxy, {
+  fastify: '4.x',
+  name: '@fastify/http-proxy'
+})
```

Comment:
```suggestion
module.exports.default = fastifyHttpProxy
module.exports.fastifyHttpProxy = fastifyHttpProxy
```

Follows the style as the other official plugins, i.e. https://github.com/fastify/fastify-rate-limit/blob/master/index.js#L279-L284

Reply:
Great thanks for your advice 😊 I have just made the following modification taking the example of the plugin

Reply:
But I keep get some error on tests suite 🤔